tools/ci/azure/cleanup_win10.yml

```diff
@@ -50,7 +57,41 @@ steps:
         Restart-Computer -Force
       }
     }
+    if (($stableExitCode -ne $exitCodeSuccess) -and
+        ($canaryExitCode -ne $exitCodeSuccess) -and
+        ($devExitCode -ne $exitCodeSuccess) -and
+        ($betaExitCode -ne $exitCodeSuccess)) {
+      Copy-Item -Path "$env:temp\*edge*.log" -Destination $(Build.ArtifactStagingDirectory) -Force
+      Throw "Failed to uninstall Edge."
+    } else {
+      Write-Host "Successfully uninstalled all Edge channels"
+    }
   displayName: 'Restore hosts file and cleanup test machine'
   condition: always()
   errorActionPreference: silentlyContinue
   ignoreLASTEXITCODE: true
+
+- powershell: |
+    $channelFolders = @('Edge', 'Edge SxS', 'Edge Dev', 'Edge Beta')
```

Comment:
Maybe just `$channels`? These aren't actually folder names as clear inside the loop, but the suffix.

Reply:
Fixed